Apache-Arrow-Flight-Tester/main.py

```diff
@@ -90,6 +90,42 @@ def list_actions(flight_client):
     print(list(response))
 
 
+def update_object_store_to_s3(flight_client, endpoint, bucket_name, access_key_id, secret_access_key):
+    object_store_type_bytes = str.encode("s3")
+    object_store_type_size = len(object_store_type_bytes).to_bytes(2, byteorder="big")
```

Comment:
Unclear why two bytes is used for the length instead of one when `len("s3") == 2` and `len("azureblobstorage") == 16` and 2<sup>8</sup> is 256

Reply:
Two bytes is used for the length so it is consistent with the way the arguments are encoded in the array of bytes.